endpoints/events/vtrack.go

```diff
@@ -49,11 +53,12 @@ type CacheObject struct {
 
 func NewVTrackEndpoint(cfg *config.Configuration, accounts stored_requests.AccountFetcher, cache prebid_cache_client.Client, bidderInfos config.BidderInfos, me metrics.MetricsEngine) httprouter.Handle {
 	vte := &vtrackEndpoint{
-		Cfg:           cfg,
-		Accounts:      accounts,
-		BidderInfos:   bidderInfos,
-		Cache:         cache,
-		MetricsEngine: me,
+		Cfg:                 cfg,
+		Accounts:            accounts,
+		BidderInfos:         bidderInfos,
+		Cache:               cache,
+		MetricsEngine:       me,
+		normalizeBidderName: openrtb_ext.NormalizeBidderName,
```

Comment:
Why are you storing a reference to `openrtb_ext.NormalizeBidderName` in this struct? Is it ever the case that you will use a different normalize function here?

Reply:
added  normaliseBidderName because it makes easier to setup tests. Followed similar pattern used in https://github.com/prebid/prebid-server/pull/3187#discussion_r1349320543

Reply:
Ahh, I see. It allows for fake bidder names to be used without throwing an error.